packages/jest-worker/src/workers/__tests__/ChildProcessWorker.test.js

```diff
@@ -63,7 +61,7 @@ it('passes fork options down to child_process.fork, adding the defaults', () =>
   });
 
   expect(childProcess.fork.mock.calls[0][0]).toBe(child);
-  expect(childProcess.fork.mock.calls[0][1]).toEqual({
+  expect(childProcess.fork.mock.calls[0][2]).toEqual({
```

Comment:
Wait what :D

Reply:
See https://github.com/facebook/jest/pull/7853/files#diff-f9414f0ef9e6230a02ef6a47be731f55R55

The typings doesn't allow skipping the `args` array. It's not mentioned in the docs, so probably fine

Reply:
Oh, totally missed the added `[]` ^^